apps/web/src/components/auth/LoginForm.tsx

```diff
@@ -108,19 +113,21 @@ export function LoginForm({}: Props) {
           placeholder="Type your password..."
           data-test-id="password"
         />
-        <Link to="/auth/reset/request">
-          <Text my={30} gradient align="center">
-            Forgot Your Password?
-          </Text>
-        </Link>
+        {!isFromVercel && (
```

Comment:
Decided to hide forgot password during `vercel onboarding`, as it would be difficult to manage the vercel configurations through the forget password flow